src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -349,6 +329,244 @@ std::vector<std::map<std::string, std::string>> filterAdditionalConfig() {
     }
 }
 
+// 3D
```

Comment:
How many new test-cases we add here? I'd like to minimize impact on functional test pass time

Reply:
624 cases are added for 3D smoke test, which toke 8.5s. I have mitigated the number by 1) move non-zero pad to post-commit, which is not common. 2) reduce input shapes enumeration. Now there are **96 cases, taking 1.3s** for smoke case. 502 cases for post-commit, take 6.8s. @Dmitry, Do you think it is acceptable? thanks!